cypress/fixtures/docs/dynamicObjects.md

```diff
@@ -0,0 +1,293 @@
+  - **String Input without Pattern**: If the string does not contain the '{{' pattern (e.g., `resolveAtRuntime("value")`), the function directly returns the value from the `runtime` environment variable.
+  - **Array of Strings**: When an array of strings is provided, the function iterates over each string, replacing the patterns with the actual values from the `runtime` environment variable.
+
+- **Options:** The function has the support to accespts the optional parameter `uppercaseFirstChar` or `lowercaseFirstChar` to convert the first character of the resolved value to uppercase or lowercase. This parameter is separated by a dot from the attribute name. This parameter is used to convert the attribute value to the correct format for the method name. For example, `resolveAtRuntime('manage_closedcaptions.set{{attribute.uppercaseFirstChar}}')`.
+
+Usage of this function by dynamic firebolt object ensures that all necessary values are dynamically resolved during test case execution in the corresponding glue code.
+
+### Examples:
+
+Assuming runtime environment variable having below details
+
+```
+runtime: {
+ attribute: 'fontSize',
+ value: 1.5,
+}
+```
+
+#### Example 1:
+
+```
+resolveAtRuntime(["result.{{attribute}}", "result.styles.{{attribute}}"])
+returns: ['result.fontSize', 'result.styles.fontSize']
+```
+
+#### Example 2:
+
+```
+resolveAtRuntime("manage_closedcaptions.set{{attribute.uppercaseFirstChar}}")
+returns: "manage_closedcaptions.setFontSize"
+```
+
+#### Example 3:
+
+```
+resolveAtRuntime("value")
+returns: 1.5
+```
+
+## Usage
+
+The dynamic calls Glues can be used in test cases along with fireboltCall dynamic objects.
+
+### Below are examples illustrating various ways to utilize dynamic firebolt objects and glue codes within test cases
+
+#### Example 1:
+
+**Firebolt object**
+
+```
+exports.ACCESSIBILITY_CLOSEDCAPTIONS_SETTINGS = {
+  method: 'accessibility.closedCaptionsSettings',
+  params: {},
+  validationJsonPath: resolveAtRuntime(['result.{{attribute}}', 'result.styles.{{attribute}}']),
+  setMethod: resolveAtRuntime('manage_closedcaptions.set{{attribute.uppercaseFirstChar}}'),
+  setParams: resolveAtRuntime('value'),
+  setValidationJsonPath: 'result',
+  setContent: null,
+  event: 'accessibility.onClosedCaptionsSettingsChanged',
+  eventValidationJsonPath: resolveAtRuntime([
+    'eventResponse.{{attribute}}',
+    'eventResponse.styles.{{attribute}}',
+  ]),
+  content: {
+    data: [
+      {
+        type: 'fixture',
+        validations: [
+          {
+            mode: 'staticContentValidation',
+            type: resolveAtRuntime('value'),
+            description: resolveAtRuntime(
+              'Validating that accessibility.closedCaptionSettings {{attribute}} is {{value}}'
+            ),
+          },
+        ],
+      },
+    ],
+  },
+};
+```
+
```

Comment:
what is the difference between Test case 1 and Test case 2 ? Looks like both are same with test case 2 having additional scenarios

Reply:
testcase 2 to shows we can use multiple api's using single object